brainstormforce/ultimate-addons-for-gutenberg
# Fix: Backward compatibility for tabs and inline notice.

### Description


### Screenshots


### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Create Inline notice and Tabs in Master.
- Switch to controls-ui-ux and check if every setting is added to a specific block after regenerate assets.

### Checklist:
